pennylane/devices/default_qubit_torch.py

```diff
@@ -257,17 +287,3 @@ def sample_basis_states(self, number_of_states, state_probability):
         return super().sample_basis_states(
             number_of_states, state_probability.cpu().detach().numpy()
         )
-
-    def _apply_operation(self, state, operation):
```

Comment:
This method was removed because:
* The following logic was moved to `execute` (as per the #1927)
```python
        if state.device != self._torch_device:
            state = state.to(self._torch_device)
```
* The rest of the method is just a simple call on the parent method, hence it can be removed to let inheritance take care of it.

Reply:
Thanks for the explanation!